tests/test_mesh_methods.py

```diff
@@ -0,0 +1,227 @@
+__doc__ = """ Test mesh class's methods in Elastica """
+
+from mesh.mesh_initializer import Mesh
+import numpy as np
+from numpy.testing import assert_allclose
+from sys import platform
+
+
+"""
+A dummy cube mesh stl file is used for testing at tests/cube.stl
+This dummy file was created using the open source code for creating meshes using 'numpy-stl'
+in numpy-stl documentation (https://numpy-stl.readthedocs.io/en/latest/usage.html#initial-usage)
+"""
+
+
+def cube_mesh_init():
+    """
+    This function initializes a new cube mesh.
+    """
+    if platform == "win32":
+        path = r"tests\cube.stl"
+    else:
+        path = r"tests/cube.stl"
+
+    mockmesh = Mesh(path)
+    return mockmesh
+
+
+"""def test_visualize():
+    This function tests the visualization of the mesh.
+    Running this test will open a new window with the mesh visualization.
+    mockmesh = cube_mesh_init()
+    try:
+        mockmesh.visualize()
+    except:
+        raise RuntimeError("Visualization failed")"""
```

Comment:
why this commented?

Reply:
Running this test actually opens up a pyvista plotter window mid-tests and it has to be closed manually to proceed further; I am doubtful how this would operate with github's automated tests.